src/state/transactions/updater.tsx

```diff
@@ -109,7 +109,9 @@ export default function Updater(): null {
                 return
               }
 
-              const isFulfilled = orderMetadata.status === GnosisProtocolTradeOrderStatus.FULFILLED
+              const isFulfilled = orderMetadata.status === 'fulfilled'
+
+              console.log(orderMetadata)
```

Comment:
we don't need it

Reply:
[6ed22de](https://github.com/levelkdev/swapr-dapp/pull/1280/commits/6ed22de5368f9372fd2fc6d703d13e4a65f669df)